detect_secrets/plugins/discord.py

```diff
@@ -0,0 +1,17 @@
+"""
+This plugin searches for Discord Bot Token
+"""
+import re
+
+from .base import RegexBasedDetector
+
+
+class DiscordBotTokenDetector(RegexBasedDetector):
+    """Scans for Discord Bot token."""
+    secret_type = 'Discord Bot Token'
+
+    denylist = [
+        # Discord Bot Token (MXXXXXXXXXXXXXXXXXXXXXXX.XXXXXX.XXXXXXXXXXXXXXXXXXXXXXXXXXX)
```

Comment:
```suggestion
        # Discord Bot Token ([M|N]XXXXXXXXXXXXXXXXXXXXXXX.XXXXXX.XXXXXXXXXXXXXXXXXXXXXXXXXXX)
```
This would make it explicit that a token may start with `M` or `N` without having to dive into the regex